exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,17 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function createShoutyGreeting(name) { 
+
+    return "HELLO " + name.toUpperCase();
+
+}
+
+console.log(createShoutyGreeting(mentor1));
```

Comment:
On this exercise we supposed to write 2 functions and nest them. Would you like to re-write?